benchmarks/java/src/main/java/com/google/protobuf/ProtoBench.java

```diff
@@ -66,17 +70,26 @@ public static void main(String[] args) {
       System.exit(1);
     }
     boolean success = true;
+    int endIndex = args.length;
     for (int i = 0; i < args.length; i++) {
-      success &= runTest(args[i]);
+      if (args[i].equals("--")) {
+        endIndex = i;
+        break;
+      }
+    }
+    for (int i = 0; i < endIndex; i++) {
+      success &= runTest(args, i, endIndex);
     }
     System.exit(success ? 0 : 1);
   }
+  
  
   /**
    * Runs a single test with specific test data. Error messages are displayed to stderr, 
    * and the return value indicates general success/failure.
    */
-  public static boolean runTest(String file) {
+  public static boolean runTest(String[] args, int currentIndex, int endIndex) {
```

Comment:
What's the meaning of these parameters?

Reply:
The parameter is for split the caliper options and the input files. 
And renamed the parameters,